mothership-ec/cog-mothership-file-manager
# soft deletes on files

#### What does this do?

Adds soft deletes to files
#### How should this be manually tested?

Make sure it doesn't break things after file deletion. Make sure the files are marked as deleted rather than actually removed in the database.
#### Related PRs / Issues / Resources?
#### Anything else to add? (Screenshots, background context, etc)
